core/cap-0045.md

```diff
@@ -311,91 +320,77 @@ then we should acknowledge that no prices at all is probably just as useful.
 ### Demand from a liquidity pool
 Consider a liquidity pool holding assets `X` and `Y` (we will also use `X` and
 `Y` to denote the reserves of those assets). Suppose the market price ratio is
-`p = p(X) / p(Y)`. To satisfy the liquidity pool budget constraint, we must
-have `pX + Y = pX' + Y'`. If `p < (1 - F) Y/X`, then the liquidity pool will
-buy `X` and sell `Y` until `p = (1 - F) Y'/X'`. If `1/p < (1 - F) X/Y`, then
-the liquidity pool will sell `X` and buy `Y` until `1/p = (1 - F) X'/Y'`.
+`p = p(X) / p(Y)`. 
 
-The budget constraint and marginal price constraint can be combined. If
-`p < (1 - F) Y/X` then
-```
-X' = (pX + Y) (1 - F) / ((2 - F) * p)
-Y' = (pX + Y) / (2 - F) .
-```
-If `1/p < (1 - F) X/Y` then
-```
-X' = (pX + Y) / ((2 - F) * p)
-Y' = (pX + Y) (1 - F) / (2 - F) .
-```
-In all other cases, the liquidity pool does not trade so
-```
-X' = X
-Y' = Y .
-```
+The liquidity pool starts at equilibrium point `E`, with reserve amounts `X` and `Y` respectively. 
 
-We will now show that the liquidity pool does take the actions described above.
-First, suppose that `p < (1 - F) Y/X`.  
-Then the liquidity pool sells `Y` and buys `X` until
-`p = (1 - F) Y'/X'`.  Combining this equation with the budget constraint gives
-`pX + Y = Y' + (1 - F)Y'`.
-Rearranging gives `Y' = (pX + Y) / (2 - F)` and `X' = (pX + Y) (1 - F) / (p * (2 - F))`.
+The spot price of asset `X` is the amount of asset `Y` exchanged for infinitesimal quantity of `X`:
 
-Observe that in this case, 
+<div align="center"><img style="background: white;" src="https://render.githubusercontent.com/render/math?math=p%20%3D%20%5Clim_%7B%5Cdelta%7Bx%7D-%3E0%7D%5Cfrac%7B%5Cdelta%7By%7D%7D%7B%5Cdelta%7Bx%7D%7D"></div>
```

Comment:
The arrow in the subscript of the limit isn't rendering correctly. If this is LaTeX then you would want to use `\to`. Note, this happens everywhere you used this notation although I'll only comment here.